tests/e2e/spec/main.spec.js

```diff
@@ -64,106 +51,100 @@ describe('Main Features', function () {
   });
 
   describe('Ad-Blocking', function () {
-    beforeEach(() => updatePrivacySettings('ad-blocking', false));
+    after(() => setPrivacyToggle('ad-blocking', true));
 
     const SELECTOR = 'ad-slot';
 
     it('does not block ads on a page', async function () {
+      await setPrivacyToggle('ad-blocking', false);
       await browser.url(PAGE_URL);
       await expect($(SELECTOR)).toBeDisplayed();
     });
 
     it('blocks ads on a page', async function () {
-      await updatePrivacySettings('ad-blocking', true);
+      await setPrivacyToggle('ad-blocking', true);
 
       await browser.url(PAGE_URL);
       await expect($(SELECTOR)).not.toBeDisplayed();
     });
   });
 
   describe('Anti-Tracking', function () {
-    beforeEach(() => updatePrivacySettings('anti-tracking', false));
+    after(() => setPrivacyToggle('anti-tracking', true));
 
     const TRACKER_IDS = ['facebook_connect', 'pinterest_conversion_tracker'];
 
     it('does not block tracker requests on the page', async function () {
-      await disableCache();
+      await setPrivacyToggle('anti-tracking', false);
       await browser.url(PAGE_URL);
 
       await switchToPanel(async function () {
         await getExtensionElement('button:detailed-view').click();
 
         for (const trackerId of TRACKER_IDS) {
-          const trackerEl = await getExtensionElement(
-            `button:tracker:${trackerId}`,
-          );
-
-          if (trackerEl.isDisplayed()) {
-            await expect(
-              getExtensionElement(`icon:tracker:${trackerId}:blocked`),
-            ).not.toBeDisplayed();
-            await expect(
-              getExtensionElement(`icon:tracker:${trackerId}:modified`),
-            ).not.toBeDisplayed();
-          }
+          await expect(
+            getExtensionElement(`icon:tracker:${trackerId}:blocked`),
+          ).not.toBeDisplayed();
+          await expect(
+            getExtensionElement(`icon:tracker:${trackerId}:modified`),
+          ).not.toBeDisplayed();
         }
       });
     });
 
     it('blocks tracker requests on the page', async function () {
-      await updatePrivacySettings('anti-tracking', true);
+      await setPrivacyToggle('anti-tracking', true);
```

Comment:
maybe disable all languages here?

Reply:
I don't understand this one; it is a test for anti-tracking. There is no relation to languages here. Did you mean regional filters?

Reply:
The config set the default language to `en-GB`, so the regional filters never have predefined regions (we don't have `en` region).

Reply:
Yes, I've meant the regional filters, I've put the comment on the wrong line - sorry.